.goreleaser.yml

```diff
@@ -63,6 +63,26 @@ builds:
         - cmd: make download-licenses
         - cmd: scripts/gon_filepath_editor.sh {{ .Path }} amd64
         - cmd: gon dist/gon_confluent_amd64.hcl
+  - id: confluent-darwin-amd64-homebrew
+    binary: confluent
+    main: cmd/confluent/main.go
+    flags:
+      - -mod=readonly
+    ldflags:
+      - -s -w -X main.version={{.Env.VERSION}} -X main.commit={{.ShortCommit}} -X main.date={{.Date}} -X main.disableUpdates=true
+    gcflags:
+      - all=-trimpath={{.Env.HOME}}/git
+    asmflags:
+      - all=-trimpath={{.Env.HOME}}/git
+    goos:
+      - darwin
+    goarch:
+      - amd64
+    hooks:
+      post:
+        - cmd: make download-licenses
```

Comment:
```suggestion
```

`confluent-darwin-amd64` post hook already runs this.

Reply:
Good catch, forgot we only need to run this once.